install-panel.sh

```diff
@@ -590,16 +590,45 @@ function firewall_firewalld {
   echo "* Firewall-cmd installed"
   print_brake 70
 }
+function centos_based_letsencrypt {
+    if [ "$OS_VER_MAJOR" == "7" ]; then
+      # Enable EPEL falowing official guide https://fedoraproject.org/wiki/EPEL
+      yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+      subscription-manager repos --enable "rhel-*-optional-rpms" --enable "rhel-*-extras-rpms"  --enable "rhel-ha-for-rhel-*-server-rpms"
 
+      # Install certbot 
+      yum install certbot
+    elif [ "$OS_VER_MAJOR" == "8" ]; then
+      # Enable EPEL falowing official guide https://fedoraproject.org/wiki/EPEL
+      dnf install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
```

Comment:
I am unsure if a conflict occurs here since the function `centos8_dep` also installs some form of epel-release package.